packages/location-field/src/index.js

```diff
@@ -593,7 +594,7 @@ class LocationField extends Component {
             aria-label="Clear location"
             onClick={this.onClearButtonClick}
           >
-            <Times size={13} />
+            <ClearButtonIcon />
```

Comment:
this line needs to look like this
```js
{ClearButtonIcon}
```

The idea is that you would instantiate the Icon component before you pass it in a as prop, that gives the user of the `location-field` component the control to pass it whatever props they want to the icon component which can allow custom styling that otp-ui won't have to worry about.  You did this with the default value, so when the default is used you trying to instantiate an already instantiated componet